src/status_im/ui/screens/wallet/choose_recipient/events.cljs

```diff
@@ -105,8 +105,8 @@
          new-amount                             (:value details)
          new-gas                                (:gas details)
          symbol-changed?                        (and old-symbol new-symbol (not= old-symbol new-symbol))]
-     (cond-> {:db         db
-              :dispatch   [:navigate-back]}
+     (cond-> {:db db}
+       (not= :deep-link origin) (assoc :dispatch [:navigate-back])
```

Comment:
what is this change  about?

Reply:
Wallet flow hardcode some `navigate-back` that are not wanted here.

Reply:
can you comment it for the glorious day we'll refactor wallet flow?

Reply:
:+1: